weather.gismeteo/resources/libs/gismeteo.py

```diff
@@ -0,0 +1,252 @@
+# -*- coding: utf-8 -*-
+# License: GPL v.3 https://www.gnu.org/copyleft/gpl.html
+
+from __future__ import unicode_literals
+from future.utils import PY26
+
+import time
+import calendar
+import requests
+
+import xml.etree.cElementTree as etree
+try:
+    etree.fromstring('<?xml version="1.0"?><foo><bar/></foo>')
+except TypeError:
+    import xml.etree.ElementTree as etree
+
+__all__ = ['GismeteoError', 'GismeteoClient']
+
+
+class GismeteoError(Exception):
+
+    pass
+
+
+class GismeteoClient(object):
+
+    _base_url = 'https://services.gismeteo.ru/inform-service/inf_chrome'
+
+    def __init__(self, lang='en'):
+
+        self._lang = lang
+
+        self._client = requests.Session()
+        if PY26:
```

Comment:
Why only PY26? Would it not be sufficient to check PY2 or PY3?

Reply:
I had an error with SSL certificate on my old android box with Kodi 16 and python 2.6, so I add check for this version. But this code actual for all python versions, so I will remove this check.